src/lib/callApi.js

```diff
@@ -34,14 +35,18 @@ export const getLocalData = (query = undefined) => {
   let totalContributions = 0;
   let totalDistance = 0;
   Object.keys(localData).forEach((key, index) => {
-    const { contributions, distance, username } = localData[key];
+    const { contributions, distance } = localData[key];
```

Comment:
looks like the code around line 40 and line 80 are very similar, maybe DRY some? (https://www.softwareyoga.com/is-your-code-dry-or-wet/)